rest/oidc_api.go

```diff
@@ -259,3 +310,19 @@ func (h *handler) getOIDCCallbackURL(providerName string, isDefault bool) string
 	}
 	return callbackURL
 }
+
+// setStateCookie sets the state cookie.
+func setStateCookie(res http.ResponseWriter, value string, httpOnly bool) {
+	http.SetCookie(res, &http.Cookie{
+		Name: stateCookieName, Value: value, Path: "/",
+		HttpOnly: httpOnly, Expires: time.Now().Add(stateCookieTimeout),
+	})
+}
+
+// deleteStateCookie deletes the state cookie.
+func deleteStateCookie(res http.ResponseWriter, httpOnly bool) {
+	http.SetCookie(res, &http.Cookie{
+		Name: stateCookieName, Value: "",
+		Expires: time.Unix(0, 0), Path: "/", HttpOnly: httpOnly,
```

Comment:
What's the reason to set Value and Expires here, if they are already the default value for http.Cookie?